sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -19,11 +19,20 @@ public enum ServiceVersion
             V2021_11_01_Preview = 1,
         }
     }
+    public partial class UploadFailedArgs : Azure.SyncAsyncEventArgs
+    {
+        internal UploadFailedArgs() : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public System.Exception Exception { get { throw null; } }
+        public int FailedLogs { get { throw null; } }
```

Comment:
This property name reads like a collection of the logs.  I think we should use `FailedLogCount` or similar to better describe what it's communication.